spec/models/labels/plate_label_cellaca_qc_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Labels::PlateLabelCellacaQc, type: :model do
+  it { expect(described_class).to be < Labels::Base }
+
+  describe '#attributes' do
+    subject(:qc_attributes) { label.qc_attributes }
+    let(:label) { described_class.new(labware) }
+
+    context 'when creating the label of a full plate' do
+      let(:labware) { create :v2_plate, pool_sizes: [96] }
```

Comment:
Aside: Pooling question: The current multistamp assigns a new pool number for each source plates wells, e.g. multistamp from 3 sources get 3 pools. For cardinal we have up to 96 tubes, so does that equal 96 distinct pools? Or is the whole pool thing irrelevant at the tube arraying stage as we've no submissions yet?

Reply:
Yep, pool numbers are a symptom of submission, so are irrelevant here.I'm not quite sure what you mean by multi-stamp assigning pool numbers though?